recce/cli.py

```diff
@@ -249,7 +253,9 @@ def diff(sql, primary_keys: List[str] = None, keep_shape: bool = False, keep_equ
 @click.option("--port", default=8000, show_default=True, help="The port to bind to.", type=int)
 @click.option("--lifetime", default=0, show_default=True, help="The lifetime of the server in seconds.", type=int)
 @click.option("--review", is_flag=True, help="Open the state file in the review mode.")
-@click.option("--api-token", help="The token used by Recce Cloud API.", type=click.STRING, envvar="RECCE_API_TOKEN")
+@click.option(
+    "--api-token", help="The API token used by Recce Cloud Share Link.", type=click.STRING, envvar="RECCE_API_TOKEN"
```

Comment:
Hi Kent,
Would "used by Share Link" be limited to the usage?
What do you think "The personal token generated by Recce Cloud"?

Reply:
Good idea!